simtools/simtel/simtel_light_emission.py

```diff
@@ -0,0 +1,552 @@
+
+            # further optional properties not used here:
+            # 'laser_external_trigger', 'laser_pulse_exptime',
+            # 'laser_pulse_offset' 'laser_pulse_twidth'
+
+            command += f" --atmosphere {self.output_directory}/model/"
+            command += f"{self._telescope_model.get_parameter_value('atmospheric_profile')}"
+        command += f" -o {self.output_directory}/{self.le_application[0]}.iact.gz"
+        command += "\n"
+
+        return command
+
+    def _make_simtel_script(self, **kwargs):  # pylint: disable=unused-argument
+        """Return the command to run simtel_array."""
+
+        # LightEmission
+        command = f"{self._simtel_source_path.joinpath('sim_telarray/bin/sim_telarray/')}"
+        command += f" -c {self._telescope_model.get_config_file()}"
+
+        def remove_line_from_config(file_path, line_prefix):
+            with open(file_path, "r", encoding="utf-8") as file:
+                lines = file.readlines()
+
+            with open(file_path, "w", encoding="utf-8") as file:
+                for line in lines:
+                    if not line.startswith(line_prefix):
+                        file.write(line)
+
+        remove_line_from_config(self._telescope_model.get_config_file(), "array_triggers")
+
+        command += " -DNUM_TELESCOPES=1"
+        command += " -I../cfg/CTA"
+        command += "iobuf_maximum=1000000000"
+        command += super()._config_option(
+            "altitude", self._site_model.get_parameter_value("corsika_observation_level")
+        )
+        # command += super()._config_option("maximum_telescopes", "1")
+        # command += super()._config_option("trigger_telescopes", "1")
+        command += super()._config_option(
+            "atmospheric_transmission",
+            self._telescope_model.get_parameter_value("atmospheric_transmission"),
+        )
+        # command += super()._config_option("trigger_current_limit", "1e10")
+        # command += super()._config_option("show", "all") # for debugging
+        # command += super()._config_option("random_state", "none")
+        # command += super()._config_option("ONLY_TRIGGERED_TELESCOPES", "0")
+        # command += super()._config_option("ONLY_TRIGGERED_ARRAYS", "0")
+        command += super()._config_option("TRIGGER_CURRENT_LIMIT", "20")
+        command += super()._config_option("TRIGGER_TELESCOPES", "1")
+        # command += super()._config_option("ARRAY_TRIGGER", "NONE")
+        command += super()._config_option("TELTRIG_MIN_SIGSUM", "7.8")
+
+        command += super()._config_option("PULSE_ANALYSIS", "-30")
+        # command += super()._config_option("SUM_BEFORE_PEAK", "-30")
+        # command += super()._config_option("ARRAY_WINDOW", "1000")
+
+        if "real" in self.default_le_config["x_pos"]:
+            _, angles = self.calibration_pointing_direction()
+            angle_theta = 180 - angles[0]
```

Comment:
I am not entirely sure about the coordinates (maybe the meaning of theta/phi should be documented in one of the docstrings). Are you sure that for both angles you need to do 180 - ...?